front/prompt/global_agent_helper_prompt.md

```diff
@@ -172,61 +251,75 @@ GPT-4 can use basic real-time data like date and time, and can give output in mu
+
+### How to create custom apps?
+
+As an Admin or a builder, to create Dust custom apps go to ⚙️ > `Developers Tools` > select `Create App` . From there, you can give the app a name and decide who can access the app. 
+
+To learn how to develop an app you can explore Dust technical documentation here - https://docs.dust.tt/ 
 
-# Troubles using Dust and limitations of the assistants
+## Troubles using Dust and limitations of the assistant
 
 ### I haven’t received a login or I am having trouble logging in
 
-If you experience issues logging in please send a message to your workspace admin or our team team@dust.tt will investigate.
+If you experience issues logging in please send a message to your workspace Admin or our team team@dust.tt will investigate.
 
-### \***\*The assistants are producing links that don’t work and falsely claiming something that’s not true. What’s going on?\*\***
+### ****The assistant is producing links that don’t work and falsely claiming something that’s not true. What’s going on?****
 
-Assistants can sometimes overstate their abilities. Despite what they might imply, assistants can't use the internet or any tools or software not approved by the admin. They can only use approved data sources and provide text responses.
+Assistants can sometimes overstate their abilities. Despite what they might imply, assistants can't use the internet or any tools or software not approved by the Admin. They can only use approved data sources and provide text responses.
 
-GPT4 and Claude are transformer-based models. They're trained to predict the next word in a sentence using probability. For example, if you input 'chair', it predicts the next word based on patterns. But it doesn't really "understand" what a chair is. That's why the assistants might sometimes make mistakes or "hallucinate".
+GPT4 and Claude are transformer-based models. They're trained to predict the next word in a sentence using probability, not grammar. For example, if you input 'chair', it predicts the next word based on patterns. But it doesn't really "understand" what a chair is. That's why the assistant might sometimes make mistakes or "hallucinate".
 
-### Why don't the assistants remember what I said earlier in a conversation?
+### Why doesn't the assistant remember what I said earlier in a conversation?
 
-Right now, the assistants can only keep track of context for a limited number of questions - 8 to be exact. After that, they might give inaccurate responses or 'hallucinate'. But as Dust keeps improving, the assistants will get better at handling longer conversations.
+Right now, your assistants can only keep track of context for a limited number of questions - 8 to be exact. After that, they might give inaccurate responses or 'hallucinate'. But as Dust keeps improving, the assistants will get better at handling longer conversations.
```

Comment:
Not true anymore. We keep track of as much as possible that fit in the context size. If there is retrieval that take only the latest but that will consume quite a lot of context size otherwise we pack the context size with as many messages as possible.